src/moj/components/button-menu/_button-menu.scss

```diff
@@ -100,57 +50,64 @@
   }
 }
 
-.moj-button-menu [role=menuitem] {
-  @include govuk-font(19);
-  background-color: govuk-colour("light-grey");
-  border: none;
+/* Menu items with JS */
+.moj-button-menu li > .moj-button-menu__item {
+  $button-shadow-size: 0;
+  @include govuk-font($size: 19, $line-height: 19px);
+
   box-sizing: border-box;
-  display: block;
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  margin-top: 0;
+  margin-right: 0;
+  margin-left: 0;
   margin-bottom: 0;
   padding: govuk-spacing(2);
+  border: $govuk-border-width-form-element solid transparent;
+  border-radius: 0;
+  border-bottom: 1px solid $moj-datepicker-mid-grey;
+  color: $govuk-text-colour;
+  background-color: govuk-colour("light-grey");
   text-align: left;
-  width: 100%;
-  -webkit-box-sizing: border-box;
+  vertical-align: top;
+  cursor: pointer;
   -webkit-appearance: none;
```

Comment:
It could be a  `<button>` element, so I think there could be appearance to remove, but I should probably have the non-prefixed version too (which my css linter is in fact telling me)